contracts/sdk/gumball-machine/src/generated/instructions/dispenseNftSol.ts

```diff
@@ -0,0 +1,174 @@
+/**
+ * This code was GENERATED using the solita package.
+ * Please DO NOT EDIT THIS FILE, instead rerun solita to update it or write a wrapper to add functionality.
+ *
+ * See: https://github.com/metaplex-foundation/solita
+ */
+
+import * as beet from '@metaplex-foundation/beet'
+import * as web3 from '@solana/web3.js'
+
+/**
+ * @category Instructions
+ * @category DispenseNftSol
+ * @category generated
+ */
+export type DispenseNftSolInstructionArgs = {
+  numItems: beet.bignum
+}
+/**
+ * @category Instructions
+ * @category DispenseNftSol
+ * @category generated
+ */
+export const dispenseNftSolStruct = new beet.BeetArgsStruct<
+  DispenseNftSolInstructionArgs & {
+    instructionDiscriminator: number[] /* size: 8 */
+  }
+>(
+  [
+    ['instructionDiscriminator', beet.uniformFixedSizeArray(beet.u8, 8)],
+    ['numItems', beet.u64],
+  ],
+  'DispenseNftSolInstructionArgs'
+)
+/**
+ * Accounts required by the _dispenseNftSol_ instruction
+ *
+ * @property [_writable_] gumballMachine
+ * @property [_writable_, **signer**] payer
+ * @property [_writable_] receiver
+ * @property [] willyWonka
+ * @property [] recentBlockhashes
+ * @property [] instructionSysvarAccount
+ * @property [] bubblegumAuthority
+ * @property [_writable_] nonce
+ * @property [] gummyroll
+ * @property [_writable_] merkleSlab
+ * @property [] bubblegum
+ * @category Instructions
+ * @category DispenseNftSol
+ * @category generated
+ */
+export type DispenseNftSolInstructionAccounts = {
```

Comment:
For some reason this is missing `systemProgram`?

Reply:
`systemProgram` key autogen'd by Solita (see line 120)